github/Organization.py

```diff
@@ -577,9 +582,18 @@ def create_repo(
             post_parameters["allow_rebase_merge"] = allow_rebase_merge
         if delete_branch_on_merge is not github.GithubObject.NotSet:
             post_parameters["delete_branch_on_merge"] = delete_branch_on_merge
+        if visibility is not github.GithubObject.NotSet:
+            post_parameters["visibility"] = visibility
+
         headers, data = self._requester.requestJsonAndCheck(
-            "POST", self.url + "/repos", input=post_parameters
+            "POST",
+            self.url + "/repos",
+            input=post_parameters,
+            headers={"Accept": Consts.repositoryVisibilityPreview}
```

Comment:
Or set headers when you check is visibility isn't NotSet?

Reply:
Do you mean to invert the condition? Like

```
headers=None if visibility is github.GithubObject.NotSet else  {"Accept": Consts.repositoryVisibilityPreview}
```

Of do you mean to initialise `headers, data = self._requester.requestJsonAndCheck` conditionally?

My apologies, I am fairly new to python so might be missing something obvious here  :)